integration/setup/setup.go

```diff
@@ -31,112 +29,30 @@ const (
 	credentialNamespace   = "giantswarm"
 )
 
```

Comment:
`HostPeerVPC` and `Resources` are just renamed to `installHostPeerVPC` and `installResources` and moved to the bottom. No other changes.